pkg/vmimage/data/script.sh

```diff
@@ -164,7 +164,21 @@ yum -y install \
     tree \
     find \
     WALinuxAgent \
-    yum-utils
+    yum-utils \
+    azure-mdsd \
+    azure-security \
+    azsec-monitor \
+    fluentd \
+    rubygem-fluent-plugin-systemd
+
+# openshift-audit dir ends up with wrong permissions, preemptively fix that
+mkdir -pv /var/log/openshift-audit
+chmod -v 0750 /var/log/openshift-audit
+
+# mdsd/azsecd/azmond are enabled by default (but misconfigured), disable them until configured
+find /etc/systemd -name azsecd\\* -or -name azsecmond\\* -or -name mdsd\\* -type l -delete
```

Comment:
I'd prefer that we disable the systemd units rather than delete these files, especially if the files are owned by the system packages.

Although, typically packages install their systemd service files into `/lib/systemd/` (and user overrides are installed in `/etc/systemd/`), so I'd be surprised if there was anything there; is that different on RHEL?

Reply:
Services install the files to `/lib/systemd` and `systemctl enable` creates the symlink in `/etc/systemd/system/<level>`. Every time I have pushed a PR with a systemctl call in vmimage @jim-minter says I should just create the symlink instead, I have no opinion going either way, just erred on the side of history :man_shrugging:

Reply:
systemctl disable/mask/whatever is fine here